src/app/modules/main.module.ts

```diff
@@ -2,6 +2,8 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { HelloWorldComponent } from '../components/hello-world.component';
 
+export { HelloWorldComponent };
```

Comment:
All components used in the library need to be exported in order for an app that uses it to compile with AoT.